contributing.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Contributing to the ICT Baseline Alignment Framework
+---
+# Contributing to the ICT Baseline Alignment Framework
+## How you can help
+### Developing and Evaluating Test Cases
+Need content from people doing this work. What do you need?
+
+I have a note about issuing templates. Does that belong bundled with this topic?
```

Comment:
not ready for publishing

Reply:
Already published. This file has not been changed from master.